test_team_not_like_us.py

```diff
@@ -30,31 +30,32 @@ def test_it_vat_code(self):
         self.assertGreater(len(result),0,'Result is empty')
         #check correct entity type
         self.assertEqual(result[0].entity_type, 'IT_VAT_CODE')
-        #check the score
-        self.assertEqual(result[0].score,0.5)
-
+        #Check score
+        self.assertGreaterEqual(result[0].score,0.5)
+        
         # context enhancement
         # add context word
-        test_str = test_str + 'iva'
+        test_str = 'piva ' + test_str
         result = analyze_text(test_str,['IT_VAT_CODE'])
+
         #expect a result
-        self.assertEqual(len(result),0,'Result is empty')
+        self.assertGreater(len(result),0,'Result is empty')
+
         #check correct entity type
         self.assertEqual(result[0].entity_type, 'IT_VAT_CODE')
+
         #check the score
-        self.assertEqual(result[0].score,0.05)
+        self.assertGreaterEqual(result[0].score,0.05)
```

Comment:
A context enhancement should improve the score - here you are looking for >= 0.05 where above you are looking for >= 0.5 - I would expect you to be looking for a larger minimum value than 0.5 here because the minimum score should be greater with context enhancement.